src/ws-egl.cpp

```diff
@@ -154,15 +138,14 @@ bool ImplEGL::initialize(EGLDisplay eglDisplay)
     if (wl_display_init_shm(display()) != 0)
         return false;
 
-    const char* extensions = eglQueryString(eglDisplay, EGL_EXTENSIONS);
-    if (isEGLExtensionSupported(extensions, "EGL_WL_bind_wayland_display")) {
+    if (epoxy_has_egl_extension(eglDisplay, "EGL_WL_bind_wayland_display")) {
         s_eglBindWaylandDisplayWL = reinterpret_cast<PFNEGLBINDWAYLANDDISPLAYWL>(eglGetProcAddress("eglBindWaylandDisplayWL"));
         assert(s_eglBindWaylandDisplayWL);
         s_eglQueryWaylandBufferWL = reinterpret_cast<PFNEGLQUERYWAYLANDBUFFERWL>(eglGetProcAddress("eglQueryWaylandBufferWL"));
         assert(s_eglQueryWaylandBufferWL);
     }
 
-    if (isEGLExtensionSupported(extensions, "EGL_KHR_image_base")) {
+    if (epoxy_has_egl_extension(eglDisplay, "EGL_KHR_image_base")) {
         s_eglCreateImageKHR = reinterpret_cast<PFNEGLCREATEIMAGEKHRPROC>(eglGetProcAddress("eglCreateImageKHR"));
```

Comment:
There two are defined by libepoxy, and we do not need to use `eglGetProcAddress()` for them:

```
% grep -E 'define egl(Create|Destroy)Image' /usr/include/epoxy/*
/usr/include/epoxy/egl_generated.h:#define eglCreateImage epoxy_eglCreateImage
/usr/include/epoxy/egl_generated.h:#define eglCreateImageKHR epoxy_eglCreateImageKHR
/usr/include/epoxy/egl_generated.h:#define eglDestroyImage epoxy_eglDestroyImage
/usr/include/epoxy/egl_generated.h:#define eglDestroyImageKHR epoxy_eglDestroyImageKHR
```

Reply:
:+1: